microsoft-edge/devtools-guide-chromium/whats-new/2019/12/devtools.md

```diff
@@ -257,7 +257,7 @@ You can now view the initiators and dependencies of a network request as a neste
 After [logging network activity in the Network panel](../../../network/index.md), select a resource and then go to the **Initiator** panel to view the **Request Initiator Chain**:
 
 *  The **inspected resource** is bold.  In the screenshot above, `ai.2.min.js` is the inspected resource.
-*  The resources above the inspected resource are the **initiators**.  In the screenshot above, `https://www.microsoftedgeinsider.com` is the initiator of `ai.2.min.js`.  In other words, `https://www.microsoftedgeinsider.com` caused the network request for `ai.2.min.js`.
+*  The resources above the inspected resource are the **initiators**.  In the screenshot above, `https://www.microsoftedgeinsider.com` is the initiator of `ai.2.min.js`.  In other words, `https://www.microsoftedgeinsider.com` caused the network request for `ai.2.min.js`.  (The screenshot shows an outdated URL.  The latest URL is `https://www.microsoft.com/edge/download/insider`.)
```

Comment:
orig url:
https://www.microsoftedgeinsider.com
goes to:
https://www.microsoft.com/en-us/edge/download/insider?form=MA13FJ